README.md

```diff
@@ -21,127 +47,131 @@ c2patool is a command line tool for working with C2PA [manifests](https://c2pa.o
+### Adding a manifest to an asset file
 
-The generated manifest store will also be reported in JSON format to stdout.
+To add C2PA manifest data to a file, use the `--manifest` (or `-m`) option with a manifest JSON file as the option argument and the path to the asset file to be signed.  Specify the output file as the argument to the `-o` or `--output` option.   For example:
 
 ```shell
 c2patool sample/image.jpg -m sample/test.json -o signed_image.jpg
 ```
 
-A parent file can be specified with the `-p` or `--parent` option or in the manifest definition. The parent file represents the state of the asset before any edits were made. 
+The tool generates a new manifest using the values given in the file and displays the manifest store to standard output (stdout).
+
+CAUTION: If the output file is the same as the source file, the tool will overwrite the source file. 
+
+If you do not use the  `-o` or `--output` option, then the tool will display the generated manifest but will not save it to a file.
+
+#### Specifying a parent file
+
+A _parent file_ represents the state of the image before any edits were made.  
+
+Specify a parent file as the argument to the `--parent`/`-p` option; for example:
 
 ```shell
 c2patool sample/image.jpg -m sample/test.json -p sample/c.jpg -o signed_image.jpg
 ```
 
+You can also specify a parent file in the manifest definition.
+
 #### Forced overwrite
-The tool will return an error if the output file already exists. The `-f` or `--force` flag may be used to override this behavior. Use this with caution.
+
+The tool will return an error if the output file already exists. Use the `--force` /  `-f` option to force overwriting the output file.  For example:
 
 ```shell
 c2patool image.jpg -m sample/test.json -f -o signed_image.jpg
 ```
 
-#### Manifest preview feature
+### Previewing a manifest
 
-If the output file is not specified, the tool will generate a preview of the generated manifest. This can be used to make sure you have formatted the manifest definition correctly.
+To display a preview of the generated manifest and ensure you've formatted the manifest definition correctly, provide the path to a manifest file as the argument with no other options or flags; for example:
 
 ```shell
-c2patool image.jpg -m sample/test.json
+c2patool sample/test.json
 ```
 
 ### Generating an external manifest
 
-The `-s` or `--sidecar` option puts the manifest in an external sidecar file in the same location as the output file. The manifest will have the same output filename but with a ".c2pa" extension. The output file will be copied but untouched. 
+Use the `--sidecar` (or `-s`) option to put the manifest in an external sidecar file in the same location as the output file. The manifest will have the same output filename but with a ".c2pa" extension. The tool will copy the output file but the original will be untouched. 
```

Comment:
This uses a different format to describe a long-form / short-form option pair than is used for `--force` above.

Reply:
Elsewhere in this PR you're using backticks for file format, but this one quotes ".c2pa". Is there a reason for the difference?

Reply:
Good catches.  I edited to us the `--option` / `-o` convention throughout and backticks for filename extension consistently throughout.